homeassistant/components/sensor/envirophat.py

```diff
@@ -157,38 +161,37 @@ def __init__(self, use_leds):
         self.voltage_1 = None
         self.voltage_2 = None
         self.voltage_3 = None
+        self.update()
```

Comment:
@pvizeli if I understand well, remove update from here, the `add_device` is already in `setup_platform`, and should instead add an update in the `EnvirophatSensor`'s `__initi__`, so it starts with a value on creation, right?